src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -1179,6 +1220,11 @@ FixedwingAttitudeControl::task_main()
 				_actuators.control[actuator_controls_s::INDEX_THROTTLE] = _manual.z;
 			}
 
+			// Add feed-forward from roll control output to yaw control output
+			// This can be used to counteract the adverse yaw effect when rolling the plane
+			_actuators.control[actuator_controls_s::INDEX_YAW] += _parameters.roll_to_yaw_ff * math::constrain(
```

Comment:
It is used to compensate the Dutch Roll effect: https://en.wikipedia.org/wiki/Dutch_roll
Especially relevant for swept-wing designs, for example: https://www.youtube.com/watch?v=v-hMXSmgK0s&t=132s